### PR TITLE
Fix launching SafariDriver

### PR DESCRIPTION
**Pull request by jgraham**

### CODE REVIEW

**Comment by jgraham** [comment]
r+ for gsnedders' changes, but I can't approve this.

**Review by nt1m — APPROVED** [review]
Just approving to unblock a fix to production, though someone should probably review this more in detail later on.

**Review by DanielRyanSmith — APPROVED** [review]

**Comment by gsnedders** [comment]
To link these together: this was a regression from https://github.com/web-platform-tests/wpt/pull/29089

**Pull request merged**
